examples/project-service-list-simple/README.md

```diff
@@ -0,0 +1,33 @@
+# project-service-list: Simple Example
+
+### Overview
+
+In this example, we will see how to generate [project services](https://cloud.google.com/config-connector/docs/reference/resource-docs/serviceusage/service) for `compute.googleapis.com` and `redis.googleapis.com`.
```

Comment:
The [resource name](https://cloud.google.com/service-usage/docs/reference/rest/v1/services) is service. Since its GCP projects, we call it a project service. Would `GCP project services` be more clearer?

Reply:
IMO `GCP project services` is better.

Reply:
done